GridKit/AutomaticDifferentiation/Enzyme/SparseWrapper.hpp

```diff
@@ -314,6 +315,8 @@ namespace GridKit
       template <typename ModelT, MemberFunctions function, class ScalarT, typename IdxT>
       struct InternalJacobian
       {
+        typedef typename GridKit::ScalarTraits<ScalarT>::real_type real_type;
+
```

Comment:
How about:
```suggestion
        using RealT = typename GridKit::ScalarTraits<ScalarT>::RealT;

```
This also assumes we replace `real_type` with `RealT` across the board.

Reply:
Similar at a few other places in this file.